create_assignment.py

```diff
@@ -0,0 +1,50 @@
+import sys
+from canvasapi import Canvas
+import boatswain_env as benv
+
+
+def wrangler_deco(parser):
+    parser.add_argument('course_id',
+                        type=str,
+                        help='course-id from Canvas',
+                        metavar='<course-id>',
+    )
+    parser.add_argument('assignment_name',
+                        type=str,
+                        help='assignment-name',
+                        metavar='<assignment-name>',
+    )
+    parser.add_argument('points',
+                        type=int,
+                        help='points',
+                        metavar='<points>',
+    )
+
+
+
+def main(args=None, config_path=None, verbose=True):
+    if args is None:
+        args = sys.argv[1:]
+
+    opt = benv.ParseOption(args, parse_deco=wrangler_deco, config_path=config_path)
+
+    canvas = Canvas(opt.config.canvasUrl(), opt.config.canvasToken())
+    course = canvas.get_course(opt.course_id)
+
+    new_assignment = course.create_assignment({
+        'name': opt.assignment_name,
+        'submission_types': [],
+        'notify_of_update': False,
+        'points_possible': opt.points,
+        'grading_type': 'points',
+        'description': '',
+        'published': True
+    })
+
+    new_assignment = new_assignment.edit(assignment={'post_manually': True})
```

Comment:
Just to double check, is there no way to set `post_manually` in the original creation and avoid an API call? I know it doesn't show up in the documentation, but curious if you tested this

Reply:
I'm testing on a local canvas-lms instance, where I found that `post_manually: True` as an assignment option doesn't work. This kind of made sense since on the web interface it's something you have to change in the gradebook rather than at assignment creation. 

But I'm realizing that even this second API call doesn't work -- the reason I thought it did is because when I was messing with things I accidentally set the default grade posting policy to manual. It seems like we're going to have to do something hacky if we want to set the policy, or maybe there's something I'm missing in the grade interface rather than the assignment interface.

Is there a reason why we don't set the default grading policy of the entire gradebook to manual? It seems like a much easier solution than doing something silly with the API.